cluster_test.go

```diff
@@ -114,6 +121,12 @@ func TestClusterDo(t *T) {
 		cmd := Cmd(&vgot, "GET", k)
 		require.Nil(t, c.doInner(cmd, stub16k.addr, k, false, 2))
 		assert.Equal(t, v, vgot)
+		assert.Equal(t, trace.ClusterRedirected{
```

Comment:
Leftovers from my `TRYAGAIN` changes, but since they don't hurt and help testing the tracing I decided too keep these.